### PR TITLE
Added React based Project

### LINKED ISSUE #2969: Adding React based productivity tool
**Describe the project you want to add with tech stack**
I'd like to add my React based productivity tool [Pomtasker](https://pom-tasker.netlify.app/)

**Expected behavior**
PomTasker is a productivity tool that combines the Pomodoro technique with a task manager. Users can create tasks, set timers for focused work periods, and track their progress as they complete their work, while also listening to lofi music in the background!

**Screenshots (optional)**
pomtasker

pomtasker2

### PR DESCRIPTION
**Pull request by tayyab-ilyas**
# Title and Issue number 

Title - PomTasker
Issue Number - 2969

Added PomTasker a ReactJS based app

Close #2969 

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

**Comment by tayyab-ilyas** [comment]
hey @MohitGupta121 ! can you check this out? and also please don't forget to assign the Gssoc'23 label. Thanks!

File: React-JS Projects/PomTasker/package-lock.json
Comment (MohitGupta121) [inline React-JS Projects/PomTasker/package-lock.json:1]: Remove this package.lock from PR.

File: React-JS Projects/PomTasker/pomtasker.png
Comment (MohitGupta121) [inline React-JS Projects/PomTasker/pomtasker.png:1]: please remove all images from PR and upload it somewhere in cloud and use that images link in your project to load that through link.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@tayyab-ilyas PTAL

File: React-JS Projects/PomTasker/package-lock.json
Comment (tayyab-ilyas) [reply React-JS Projects/PomTasker/package-lock.json:1]: Sorry it was silly of me to push this , have fixed it!

File: React-JS Projects/PomTasker/pomtasker.png
Comment (tayyab-ilyas) [reply React-JS Projects/PomTasker/pomtasker.png:1]: I have removed the images instead added a live deployment link in the README.md.

File: React-JS Projects/PomTasker/public/vite.svg
Comment (MohitGupta121) [inline React-JS Projects/PomTasker/public/vite.svg:1]: @tayyab-ilyas also remove this

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]
@tayyab-ilyas you removed images but you also have to replace the images path with the image link from where image get load.

File: React-JS Projects/PomTasker/public/vite.svg
Comment (tayyab-ilyas) [reply React-JS Projects/PomTasker/public/vite.svg:1]: actually images were just part of the README.md, so I thought its better to remove the website screenshots and instead add the live deployment link.

Comment (tayyab-ilyas) [reply React-JS Projects/PomTasker/public/vite.svg:1]: > @tayyab-ilyas also remove this

done.

**Pull request merged**